samples/DataLake/data-lake-shore-encryption/desktop.ini

```diff
@@ -0,0 +1,2 @@
+[.ShellClassInfo]
+LocalizedResourceName=data-lake-store-encryption
```

Comment:
Hmmm something weird happened here. Looks like Windows changed the display name for the folder without actually renaming it?

Please delete this desktop.ini file and rename the folder. If it does this again, try creating the new folder, move the files, and delete old folder.